forum_leaderboard/static/html/forum_leaderboard.html

```diff
@@ -0,0 +1,8 @@
+<div class="forum_leaderboard_block">
+<h2>{{ display_name }}</h2>
+<ul>
+    {% for thread in threads %}
+        <li>{{ thread.title }} (Score: {{ thread.votes.point }})</li>
```

Comment:
Links are already working in my `sprint_surplus` branch, though I've not pushed it. I'll add usernames, too.